microsoft-edge/dualengine/reference/dualenginesessionfactory.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Factory object for DualEngine interfaces.
+title: DualEngine Win32 C++ DualEngineSessionFactory
+author: MSEdgeTeam
+ms.author: edgededev
+ms.prod: microsoft-edge
+ms.date: 11/16/2023
+keywords: dual engine, dualengine, iemode, win32 apps, win32, edge, ie mode, edge html, DualEngineSessionFactory
+topic_type: 
+- APIRef
+api_name:
+- DualEngineSessionFactory
+- DualEngineSessionFactory.GetSpecifiedVersion
+- DualEngineSessionFactory.GetVersionedBrowserSession
+- DualEngineSessionFactory.TryUnlockFeature
+api_type:
+- COM
+api_location:
+- ieframe.dll
+---
+
+# interface DualEngineSessionFactory
+
+> [!IMPORTANT]
+> The DualEngine API is a limited access feature. Contact dualengineapiaccess@microsoft.com for more information.
+
+Factory object for DualEngine interfaces.
+
+## Summary
+
+ Members                        | Descriptions
+--------------------------------|---------------------------------------------
+[GetSpecifiedVersion](#getspecifiedversion) | Returns the known DualEngine interface version, if there is one.
+[GetVersionedBrowserSession](#getversionedbrowsersession) | Gets a DualEngine Session object of the requested version.
+[TryUnlockFeature](#tryunlockfeature) | Called to unlock the DualEngine interface.
+
+## Members
+
+#### GetSpecifiedVersion
+
+Returns the known DualEngine interface version, if there is one.
+
+> public HRESULT [GetSpecifiedVersion](#getspecifiedversion)(__out UINT * version) const
+
+#### Parameters

```

Comment:
```suggestion
###### Parameters
```
global: make Parameters & Returns headings smaller than member name heading.  This repo uses h2/h4/h6 pattern, especially skipping h3 b/c practically same size as h2.  Uses h5 for tab-sets.

Reply:
demoted to h6 today locally and noted (at Returns comment) should do for both headings